editor/themes/editor_theme_manager.cpp

```diff
@@ -1736,6 +1742,8 @@ void EditorThemeManager::_populate_editor_styles(const Ref<EditorTheme> &p_theme
 				style_flat_button->set_content_margin((Side)i, p_config.button_style->get_margin((Side)i) + p_config.button_style->get_border_width((Side)i));
 			}
 
+			Ref<StyleBoxFlat> style_hover_button = p_config.button_style_hover->duplicate();
```

Comment:
```suggestion
			Ref<StyleBoxFlat> style_flat_button_hover = p_config.button_style_hover->duplicate();
```

To stay consistent with existing code.